src/memtransfer_wrapper.cu

```diff
@@ -51,6 +49,7 @@ int ALLOCGPUMEM2D_PLAN(CUFINUFFT_PLAN d_plan)
 				checkCudaErrors(cudaMalloc(&d_plan->subprobstartpts,
 						(numbins[0]*numbins[1]+1)*sizeof(int)));
 			}
+			break;
```

Comment:
C-style switch is really dumb, having to have break here. It's like the old days of "goto line ...".
Why not replace switch w/ if then else? No breaks required!